commands/moderation/tag.js

```diff
@@ -0,0 +1,73 @@
+const got = require('got')
+
+const { FILESIZE_LIMIT } = require('../../src/constants/tags.js')
+const Tag = require('../../src/models/Tag.js')
+
+exports.run = async (handler, message, args, pre) => {
+	let tag = await Tag.findOne({ guildId: message.guild.id, name: args.name }).exec()
+
+	if (args.delete) {
+		if (tag) {
+			await tag.delete()
+			return message.reply(`deleted tag "${args.name}".`)
```

Comment:
Minor issue, but I feel like we should capitalize this stuff for consistency? The other moderation commands do it so I don't see why we shouldn't here

Reply:
Only `approve` does that. All the other commands that respond in this fashion (`helper`, `languages`, and `quote`) don't capitalize the first letter. I agree that it should be consistent, but in my opinion we should change `approve` to match. (Even putting aside current majority practice, the first letter after a comma just shouldn't be capitalized.)